app/controllers/AnyMoreChangesController.scala

```diff
@@ -71,7 +71,7 @@ class AnyMoreChangesController @Inject()(appConfig: FrontendAppConfig,
       )
   }
 
-  private def dateToCompleteDeclaration: String = LocalDate.now().plusDays(appConfig.daysDataSaved).
```

Comment:
Method should only really be public if it's referred to from another class in the main code base. I'd just copy this code into the unit test that calls it.

Reply:
Updated